### PR TITLE
chore(e2e): Change the domain of fictional emails so we can cleanup user corerctly

### PR DESCRIPTION
**Pull request by octoper**
## Description

This PR updates the domain that is used for the fictional emails we are generating for Integration tests.

## Checklist

- [ ] `npm test` runs as expected.
- [ ] `npm run build` runs as expected.
- [ ] (If applicable) [JSDoc comments](https://jsdoc.app/about-getting-started.html) have been added or updated for any package exports
- [ ] (If applicable) [Documentation](https://github.com/clerk/clerk-docs) has been updated

## Type of change

- [ ] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

### CODE REVIEW

**Review by anagstef — APPROVED** [review]

File: integration/testUtils/usersService.ts [hunk 1]
```diff
@@ -67,7 +67,7 @@ export const createUserService = (clerkClient: ClerkClient) => {
         withUsername = false,
       } = options || {};
       const randomHash = hash();
-      const email = fictionalEmail ? `${randomHash}+clerk_test@example.com` : `clerkcookie+${randomHash}@mailsac.com`;
+      const email = fictionalEmail ? `${randomHash}+clerk_test@clerkcook.ie` : `clerkcookie+${randomHash}@mailsac.com`;
```
Comment (LekoArts) [inline integration/testUtils/usersService.ts:70]: Wouldn't `clerkcookie.com` also work? Then you could leave the `query` in the `cleanup.setup.ts` the same

File: integration/tests/sign-in-flow.test.ts [hunk 2]
```diff
@@ -109,8 +109,8 @@ testAgainstRunningApps({ withEnv: [appConfigs.envs.withEmailCodes] })('sign in f
     await u.po.signIn.getForgotPassword().click();
     await u.po.signIn.getResetPassword().click();
     await u.po.signIn.enterTestOtpCode();
-    await u.po.signIn.setPassword(`${fakeUserWithPasword.password}_reset`);
-    await u.po.signIn.setPasswordConfirmation(`${fakeUserWithPasword.password}_reset`);
+    await u.po.signIn.setPassword('password_reset');
```
Comment (LekoArts) [inline integration/tests/sign-in-flow.test.ts:112]: Why was this change necessary?

Comment (octoper) [reply integration/tests/sign-in-flow.test.ts:112]: Yeah it was failing as the new password which is 2 times the email was too long.

File: integration/testUtils/usersService.ts (hunk 1 above)
Comment (octoper) [reply integration/testUtils/usersService.ts:70]: We also use email as password and there is a limit on characteres so I tried to keep it short

Comment (LekoArts) [reply integration/testUtils/usersService.ts:70]: How about changing the password to `{email}{randomHash}` instead, then you could use the full `clerkcookie` name and wouldn't need to change the individual test. Better to fix the root cause than to fix it in an individual test. This way no one else will run into it.

What do you think? :)

Comment (nikosdouvlis) [reply integration/testUtils/usersService.ts:70]: Agree with Lennart here :)

Comment (LekoArts) [reply integration/testUtils/usersService.ts:70]: The hash will be inside the email name so I think you can still figure out the password 👍

**Review by LekoArts — APPROVED** [review]
